dfaligner/model.py

```diff
@@ -127,6 +128,11 @@ def check_and_upgrade_checkpoint(self, checkpoint):
             # Upgrading from 0.0 to 1.0 requires no changes; future versions might require changes
             checkpoint["model_info"]["version"] = "1.0"
 
+        # We changed the handling of phonological features in everyvoice==0.3.0
+        if ckpt_version < Version("1.1"):
+            raise ValueError(f"""There were breaking changes to the handling of text in version 1.1, introduced in version 0.3.0 of EveryVoice.
+                               Your model is version {ckpt_version} and your model will not work as a result. Please downgrade to everyvoice 0.2.0.""")
```

Comment:
0.2.0 never existed. You mean 0.2.0a1. Unless we publish 0.2.0==0.2.0a1, trying `pip install everyvoice==0.2.0` will fail. Currently, even `pip install everyvoice` fails. And `pip install everyvoice<0.3` will continue to fail, only with an explicit `pip install everyvoice==0.2.0a1` will pip ever accept to install an alpha or beta version.